src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -1,589 +1,131 @@
 package com.eppo.sdk;
 
-import cloud.eppo.ShardUtils;
-import cloud.eppo.Utils;
-import cloud.eppo.rac.Constants;
-import cloud.eppo.rac.dto.*;
-import cloud.eppo.rac.exception.EppoClientIsNotInitializedException;
-import cloud.eppo.rac.exception.InvalidInputException;
-import com.eppo.sdk.helpers.*;
-import com.eppo.sdk.helpers.bandit.BanditEvaluator;
-import com.fasterxml.jackson.databind.JsonNode;
-import java.util.*;
-import java.util.stream.Collectors;
-import org.ehcache.Cache;
+import cloud.eppo.BaseEppoClient;
```

Comment:
It's not worth comparing the diff here, just look at the new (green) code to see what's going on. I'll add comments to help guide.